### PR TITLE
Update user feedback messages for Country Filter

### LINKED ISSUE #123: Add better user feedback messages for Country Filter

### PR DESCRIPTION
**Pull request by raysonkoh**
## Description

Fixes #123 

## Testing

No tests were added.

## Remarks

NIL

### CODE REVIEW

File: src/main/java/seedu/address/model/client/ClientCountryMatchesInputCountryPredicate.java [hunk 1]
```diff
@@ -25,6 +25,15 @@ public boolean test(Client client) {
         return client.getCountry().equals(country);
     }
 
+    /**
+     * Returns string representation of user-input country.
+     *
+     * @return The string representation of the user-input country.
+     */
+    public String getInputCountryRepr() {
```
Comment (qwoprocks) [inline src/main/java/seedu/address/model/client/ClientCountryMatchesInputCountryPredicate.java]:
```suggestion
    public String getInputCountryStringRepresentation() {
```
Better to have descriptive function names without abbreviations imo

File: src/main/java/seedu/address/logic/commands/CountryFilterCommand.java [hunk 2]
```diff
@@ -12,6 +12,12 @@
 public class CountryFilterCommand extends Command {
 
     public static final String COMMAND_WORD = "country filter";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all clients whose country matches "
+            + "the specified country-code and displays them as a list with index numbers.\n"
+            + "Parameters: "
+            + PREFIX_COUNTRY + "COUNTRY_CODE\n"
```
Comment (qwoprocks) [inline src/main/java/seedu/address/logic/commands/CountryFilterCommand.java]: Probably can be on the same line for clarity

**Review by qwoprocks — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/client/ClientCountryMatchesInputCountryPredicate.java (hunk 1 above)
Comment (raysonkoh) [reply src/main/java/seedu/address/model/client/ClientCountryMatchesInputCountryPredicate.java]: Ok sure

File: src/main/java/seedu/address/logic/commands/CountryFilterCommand.java (hunk 2 above)
Comment (raysonkoh) [reply src/main/java/seedu/address/logic/commands/CountryFilterCommand.java]: Done

**Review by qwoprocks — APPROVED** [review]

**Comment by qwoprocks** [comment]
LGTM

**Pull request merged**